src/features/fireWeather/components/NoonForecastTable.tsx

```diff
@@ -61,15 +61,15 @@ const NoonForecastDisplay = ({ values, testId, title }: Props) => {
                 <TableCell>Temp (°C)</TableCell>
                 {values.map(v => (
                   <TableCell key={v.datetime}>
-                    {v.temperature.toFixed(MODEL_VALUE_DECIMAL)}
+                    {v.temperature && v.temperature.toFixed(MODEL_VALUE_DECIMAL)}
```

Comment:
what happens if v.temperature is null? Is the table cell just empty?

Reply:
Yeap you can actually check this by running cypress tests. (Look at the first column of the observed value table)